todel/src/models/logic/messages/edit.rs

```diff
@@ -114,7 +118,7 @@ impl Message {
             self.content = content;
         }
 
-        if edit.attachments.is_some() {
+        if !attachments.is_empty() {
```

Comment:
this is still not quite right- attachments being empty is an indicator that they should be emptied, revert it to `edit.attachments.is_some()` or equiv pls